### DISCUSSION TITLE
Customising endpoint for a GET request

### DISCUSSION THREAD

**Discussion by MarkOSullivan94** [serverpod/serverpod#1357] [Q&A] [answered]
In the example project we have `ExampleEndpoint` with `hello` as the method which was provided. 

Is there anyway we can produce an endpoint so that when I make a GET request to `http://localhost:8080/version` that it'll return the following JSON:

```json
{
  "result": "version 1.0.0"
}
```

If so how do I go about implementing this?

**Comment by vlidholt** [comment, marked as answer]
If you are using a Flutter app as the client, this should be done using a method and endpoint.

If you need it for some other type of use case, e.g. a webhook or so, your best option is to use the Relic web-server that is bundled together with Serverpod.
https://docs.serverpod.dev/concepts/webserver
